docs/ref/utils.txt

```diff
@@ -145,9 +145,10 @@ The functions defined in this module share the following properties:
 
     Parses a string and returns a :class:`datetime.timedelta`.
 
-    Expects data in the format ``"DD HH:MM:SS.uuuuuu"`` or as specified by ISO
-    8601 (e.g. ``P4DT1H15M20S`` which is equivalent to ``4 1:15:20``) or
-    PostgreSQL's day-time interval format (e.g. ``3 days 04:05:06``).
+    Expects data in the format ``"DD HH:MM:SS.uuuuuu"`` or ``"DD HH:MM:SS,uuuuuu"``
+    or as specified by ISO 8601 (e.g. ``P4DT1H15M20S`` which is equivalent to
+    ``4 1:15:20``) or PostgreSQL's day-time interval format
+    (e.g. ``3 days 04:05:06``).
```

Comment:
There should likely be a `::versionadded 3.1` here for the new accepted format. And maybe a mention in the release notes?

Reply:
how do I do a '::versionadded 3.1' ? I will find out how to add release notes to it.

Reply:
https://docs.djangoproject.com/en/2.2/internals/contributing/writing-documentation/#documenting-new-features